contracts/finality/src/contract.rs

```diff
@@ -104,20 +109,34 @@ pub fn execute(
 #[cfg(test)]
 pub(crate) mod tests {
     use super::*;
+    use std::marker::PhantomData;
 
+    use cosmwasm_std::testing::{MockApi, MockQuerier, MockStorage};
     use cosmwasm_std::{
         from_json,
-        testing::{message_info, mock_dependencies, mock_env},
+        testing::{message_info, mock_env},
+        OwnedDeps,
     };
     use cw_controllers::AdminResponse;
 
     pub(crate) const CREATOR: &str = "creator";
     pub(crate) const INIT_ADMIN: &str = "initial_admin";
     const NEW_ADMIN: &str = "new_admin";
 
+    pub type BabylonDeps = OwnedDeps<MockStorage, MockApi, MockQuerier, BabylonQuery>;
```

Comment:
Could you add comments to explain what you're mocking here and why?

Reply:
This is needed because of the last trait (`BabylonQuery`), which has a default of `Empty`. Will document.